CodeGeneration/Sources/generate-swiftsyntax/GenerateSwiftSyntax.swift

```diff
@@ -55,49 +72,51 @@ struct GenerateSwiftSyntax: ParsableCommand {
+        GeneratedFileSpec([swiftSyntaxDir, "generated", "raw", "RawSyntaxValidation.swift"], rawSyntaxValidationFile),
+        GeneratedFileSpec([swiftSyntaxDir, "generated", "SyntaxAnyVisitor.swift"], syntaxAnyVisitorFile),
+        GeneratedFileSpec([swiftSyntaxDir, "generated", "SyntaxBaseNodes.swift"], syntaxBaseNodesFile),
+        GeneratedFileSpec([swiftSyntaxDir, "generated", "SyntaxCollections.swift"], syntaxCollectionsFile),
+        GeneratedFileSpec([swiftSyntaxDir, "generated", "SyntaxEnum.swift"], syntaxEnumFile),
+        GeneratedFileSpec([swiftSyntaxDir, "generated", "SyntaxKind.swift"], syntaxKindFile),
+        GeneratedFileSpec([swiftSyntaxDir, "generated", "SyntaxRewriter.swift"], syntaxRewriterFile),
+        GeneratedFileSpec([swiftSyntaxDir, "generated", "SyntaxTraits.swift"], syntaxTraitsFile),
+        GeneratedFileSpec([swiftSyntaxDir, "generated", "SyntaxTransform.swift"], syntaxTransformFile),
+        GeneratedFileSpec([swiftSyntaxDir, "generated", "SyntaxVisitor.swift"], syntaxVisitorFile),
+        GeneratedFileSpec([swiftSyntaxDir, "generated", "TokenKind.swift"], tokenKindFile),
+        GeneratedFileSpec([swiftSyntaxDir, "generated", "Tokens.swift"], tokensFile),
+        GeneratedFileSpec([swiftSyntaxDir, "generated", "Trivia.swift"], triviaFile),
 
         // SwiftSyntaxBuilder
-        TemplateSpec(sourceFile: buildableCollectionNodesFile, module: swiftSyntaxBuilderDir, filename: "BuildableCollectionNodes.swift"),
-        TemplateSpec(sourceFile: buildableNodesFile, module: swiftSyntaxBuilderDir, filename: "BuildableNodes.swift"),
-        TemplateSpec(sourceFile: resultBuildersFile, module: swiftSyntaxBuilderDir, filename: "ResultBuilders.swift"),
-        TemplateSpec(sourceFile: syntaxExpressibleByStringInterpolationConformancesFile, module: swiftSyntaxBuilderDir, filename: "SyntaxExpressibleByStringInterpolationConformances.swift"),
+        GeneratedFileSpec([swiftSyntaxBuilderDir, "generated", "BuildableCollectionNodes.swift"], buildableCollectionNodesFile),
+        GeneratedFileSpec([swiftSyntaxBuilderDir, "generated", "BuildableNodes.swift"], buildableNodesFile),
+        GeneratedFileSpec([swiftSyntaxBuilderDir, "generated", "ResultBuilders.swift"], resultBuildersFile),
+        GeneratedFileSpec([swiftSyntaxBuilderDir, "generated", "SyntaxExpressibleByStringInterpolationConformances.swift"], syntaxExpressibleByStringInterpolationConformancesFile),
       ]
       + BASE_KIND_FILES.map { baseKind in
-        TemplateSpec(sourceFile: syntaxNode(emitKind: baseKind.key), module: swiftSyntaxDir, filename: "syntaxNodes/\(baseKind.value)")
-      }
+        GeneratedFileSpec([swiftSyntaxDir, "generated", "syntaxNodes", baseKind.value], syntaxNode(emitKind: baseKind.key))
+      } + [
+        GeneratedFileSpec([swiftSyntaxDir, "Documentation.docc", "generated", "SwiftSyntax.md"], swiftSyntaxDoccIndex)
```

Comment:
Nit pick and don't need to be part of this PR, personally I would have a `private let generatedPath: String = "generated"` next to  `swiftSyntaxDir`, `swiftSyntaxBuilderDir` and to so on.

Reply:
Good suggestion. I updated it.